packages/tools/f-vue-icons/src/index.js

```diff
@@ -143,6 +143,7 @@ export { default as StarFilledIcon } from '../icons/StarFilledIcon';
 export { default as StopwatchIcon } from '../icons/StopwatchIcon';
 export { default as SuccessIcon } from '../icons/SuccessIcon';
 export { default as TickIcon } from '../icons/TickIcon';
+export { default as VeganIcon } from '../icons/VeganIcon';
```

Comment:
@VasRoko If you haven't already, I would suggest using a tool such as `yalc` locally to verify that this update works as expected when you consume the package. 

Please could you confirm this before we merge?